sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/OtlpConfigUtil.java

```diff
@@ -46,6 +48,16 @@ static void configureOtlpExporterBuilder(
     String endpoint = config.getString("otel.exporter.otlp." + dataType + ".endpoint");
     if (endpoint == null) {
       endpoint = config.getString("otel.exporter.otlp.endpoint");
+      String signalPath = signalPath(dataType);
+      String protocol = getOtlpProtocol(dataType, config);
+      if (endpoint != null
+          && protocol.equals(PROTOCOL_HTTP_PROTOBUF)
+          && !endpoint.endsWith(signalPath)) {
+        if (!endpoint.endsWith("/")) {
+          endpoint += "/";
+        }
```

Comment:
I took a little liberty from the spec here by appending a `/` before the signal path if it doesn't exist. 

A strict interpretation would cause `http://localhost:4317/foo` to be transformed to `http://localhost:4317/foov1/traces`

Reply:
I think that's OK. If someone needs a path without slash (or something entirely different), they can configure it with the per-signal configuration.
Still, it should probably also be fixed in the spec.